opendocdb-cts/internal/mongosh/fixture_test.go

```diff
@@ -141,6 +156,7 @@ func TestConvertFixtures(t *testing.T) { //nolint:revive // exceeds number of li
 			},
 			expected: `
 			db.c.insertMany([{"_id": "decimal128", "v": Decimal128("42.13")}]);`,
+			skip: "https://github.com/OpenDocDB/cts/issues/34",
```

Comment:
Isn't it the same issue as https://github.com/OpenDocDB/cts/issues/22 ?

Reply:
https://github.com/OpenDocDB/cts/issues/34#issuecomment-2809886501